### PR TITLE
Gang Tweaks

### PR DESCRIPTION
**Pull request by Ikarrus**
- Gangs have a maximum of three attempts at dominations each
  - Reaching their maximum will no longer allow them to start any more dominations or recall the shuttle
- Added a couple of redone gang tag sprites by Joan
- You have to be conscious to hear gang messages now

### CODE REVIEW

**Comment by WJohn** [comment]
What happens if they hit the final attempt? Does that gang's objective change or something? Does it just disband?

**Comment by Ikarrus** [comment]
They can't start any more. They can't win.

**Comment by WJohn** [comment]
What if both gangs lose all attempts? Should the game end immediately (most if not all of the crew would be converted by now) or should it just wait until the shuttle arrives?

**Comment by Ikarrus** [comment]
From what I've seen players usually don't like to have their round end so abruptly. The shuttle, or an antag win, is usually preferred.

We're talking about an extreme edge case, though. Usually gangs don't survive the first attempt.

File: html/changelogs/Ikarrus-gang541.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - imageadd: "Added a couple of new gang tag sprites by Joan."
```
Comment (Jordie0608) [inline html/changelogs/Ikarrus-gang541.yml]: Wasn't this in your previous PR? There's a very similar message in the changelog already.

Comment (Ikarrus) [inline html/changelogs/Ikarrus-gang541.yml]: Yes. Other new ones are getting added.

Comment (Anonus) [inline html/changelogs/Ikarrus-gang541.yml]: These are resprites, however, so a different message might be better.

**Pull request merged**